datumaro/plugins/voc_format/extractor.py

```diff
@@ -58,8 +58,24 @@ def _load_categories(dataset_path):
 
     @staticmethod
     def _load_subset_list(subset_path):
-        with open(subset_path) as f:
-            return [line.split()[0] for line in f]
+        subset_list = []
+        with open(subset_path, encoding='utf-8') as f:
+            for line in f:
+                dirname = osp.basename(osp.dirname(subset_path))
```

Comment:
I think this is better to be done with check of `self._task`. These files are not required to be inside the default task lists directory - it is just a default location.